code/__DEFINES/colors.dm

```diff
@@ -20,15 +20,19 @@
 #define COLOR_OLIVE            "#808000"
 #define COLOR_LIME             "#32CD32"
 #define COLOR_GREEN            "#008000"
+#define COLOR_VIBRANTGREEN     "#00FF00"
```

Comment:
```suggestion
#define COLOR_MOSTLY_PURE_LIME_GREEN     "#00FF00"
```

Reply:
god this is even worse. doing COLOR_BRIGHT_LIME_GREEN